applications/crossbar/priv/couchdb/schemas/call_recordings.json

```diff
@@ -71,6 +71,11 @@
             "name": "from",
             "type": "string"
         },
+        "id": {
```

Comment:
please don't add `id` as a schema property, these are tracked by `_id` from Couch and leaked onto the document in the response payload. It would be weird to set "id" in your request and see the real id in the response.